garbage_classifier/pipelines/kubeflow-pipelines.py

```diff
@@ -0,0 +1,147 @@
+import uuid
+from typing import Optional
+
+import kfp
+import typer
+from kfp import dsl
+from kubernetes.client.models import V1EnvVar
+
+IMAGE = "rostyslavskliar/garbage-classifier-trainer:latest"
+
+
+@dsl.pipeline(name="garbage_classifier_traininig_pipeline", description="Pipeline for training garbage classifier from scratch")
+def garbage_classifier_traininig_pipeline(s3_access_key, s3_secret_key, s3_bucket, s3_prefix, wandb_api_key):
+
+    load_data = dsl.ContainerOp(
+        name="load_data",
+        command=["python", "garbage_classifier/cli.py", "load-train-data",
+                 s3_access_key, s3_secret_key, s3_bucket, s3_prefix],
+        image=IMAGE,
+        file_outputs={"train": "data/real-ds/train.tar.gz",
+                      "test": "data/real-ds/test.tar.gz"},
+    )
```

Comment:
try to format this code, consider to use https://github.com/psf/black

Reply:
ok